examples/alpine-slim.yaml

```diff
@@ -0,0 +1,23 @@
+contents:
+  repositories:
+    - https://dl-cdn.alpinelinux.org/alpine/edge/main
+  packages:
+    - alpine-baselayout-data
+    - apk-tools
+    - busybox
+
+entrypoint:
+  command: /bin/sh -l
+
+# optional environment configuration
+environment:
+  PATH: /usr/sbin:/sbin:/usr/bin:/bin
+
+# data for /etc/os-release if it does not already exist
+# in the image
+os-release:
+  id: alpine
+  version-id: '3.16.0'
+  name: 'Alpine Slim'
+  pretty-name: 'Alpine Slim (apko)'
```

Comment:
Can we default this, or do we need to update all the distroless repos with this block?  I’d love for things to work with scanners by default 🤞

Reply:
it has defaults, but we need to figure out what sensible ones are